python/mujincontrollerclient/controllerclientbase.py

```diff
@@ -1123,24 +1123,24 @@ def Restore(self, file, restoreconfig=True, restoremedia=True, timeout=600):
     # Debugging related
     #
 
-    def ListDebugPk(self, timeout=5):
-        """returns available debug pks from controller
+    def GetDebugResources(self, timeout=5):
+        """returns available debug resources from controller
 
         :param timeout: Amount of time in seconds to wait before failing, defaults to 5
-        :return: Available debug pks
+        :return: Available debug resources
         """
-        return self._webclient.APICall('GET', u'debug/', timeout=timeout)
+        return self.ObjectsWrapper(self._webclient.APICall('GET', u'debug/', timeout=timeout))
 
-    def GetDebugPk(self, pk, timeout=10):
-        """downloads contents of the given pk
+    def DownloadDebugResource(self, debugresource, timeout=10):
```

Comment:
The parameter `debugresource` should be named `debugresourcePk`, because it's a primary key string, not the debug resource itself.